docs/team/joseph.md

```diff
@@ -1,6 +1,33 @@
-# John Doe - Project Portfolio Page
+# Joseph Abraham - Project Portfolio Page
```

Comment:
Naming of the file should be that of your github username. After changing that i think it will be good to be merged!

Reply:
Roger, thank you!